Super_Simple_RGB_WiFi_Lamp/Super_Simple_RGB_WiFi_Lamp.ino

```diff
@@ -29,23 +29,25 @@
 #define DATA_PIN 5
 
 // Set the number of LED's - Simply count how many there are on your string and enter the number here.
-#define NUM_LEDS 66
+#define NUM_LEDS 131
 
 // Set your UTC offset - This is the time zone you are in. for example +10 for Sydney or -4 for NYC.
-#define UTC_OFFSET +10
+#define UTC_OFFSET +2
 
 // Set up LED's for each side - These arrays hold which leds are on what sides. For the basic rectangular shape in the example this relates to 4
 // sides and 4 arrays. You must subract 1 off the count of the LED when entering it as the array is 0 based. For example the first LED on the 
 // string is entered as 0.
-int topLeds[]     = {18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47};
-int bottomLeds[]  = {14, 13, 12, 11, 10, 9, 8, 7, 6, 5, 4, 3, 2, 1, 0, 65, 64, 63, 62, 61, 60, 59, 58, 57, 56, 55, 54, 53, 52, 51};
-int leftLeds[]    = {48, 49, 50};
-int rightLeds[]   = {15, 16, 17};
+int topLeds[]     = {36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95};
+int bottomLeds[]  = {29,28,27,26,25,24,23,22,21,20,19,18,17,16,15,14, 13, 12, 11, 10, 9, 8, 7, 6, 5, 4, 3, 2, 1, 0, 130,130,129,128,127,126,125,124,123,122,121,120,119,118,117,116,115,114,113,112,111,110,109,108,107,106,105,104,103,102};
+int leftLeds[]    = {96,97,98,99,100,101};
+int rightLeds[]   = {30,31,32,33,34,35};
 
 // Eneter your wifi credentials here - If you would like to enter your wifi credentials now you can with these variables. This is a nice easy 
 // method to get your ESP8266 connected to your network quickly. If you don't you can always set it up later in the wifi portal.
 String SSID = "";
 String Password = "";
+
+bool autoOnWithModeChange = false;
 // ########################################################## End of Sketch Variables ##########################################################
```

Comment:
All the changes to the base variables needs to be reverted to remain the same as my tutorial

Reply:
Can't those variables be put in a seperate file, for instance "UserSettings"? Would make it easier to maintain I think.
Also I believe the autoOnWithModeChange has to be placed there.... (putted back to true)